package practicals;
/**
 * 
 * @author Karan
 *
 *NO,There can not be a finally statement that is not attached to a try-catch block.
 *As given below if we uncomment finally block then it generates error.
 */
public class Prac_3 {

	public static void main(String[] args) {
		
		System.out.println(" given below finally block");
	/*	finally
		{
		System.out.println("Finall block");	
		}
	*/	
	}
}
